packages/fswatch_async/fswatch_async.11-0.1.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "zandoye@gmail.com"
+authors: [ "ZAN DoYe" ]
+homepage: "https://github.com/kandu/ocaml-fswatch/"
+bug-reports: "https://github.com/kandu/ocaml-fswatch/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/kandu/ocaml-fswatch"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "fswatch"
```

Comment:
Should the version of the main package be synched with this one? (e.g. are you planning to release both at the same time?)
```suggestion
  "fswatch" {= version}
```

Reply:
An old version of fswatch is satisfying. The api of fswatch is quite stable so fswatch and its extensions are associated loosely for now.